remote-api/src/main/java/org/alfresco/rest/api/impl/TagsImpl.java

```diff
@@ -279,7 +285,7 @@ public List<Tag> createTags(final StoreRef storeRef, final List<Tag> tags, final
 			.peek(tag -> {
 				if (parameters.getInclude().contains(PARAM_INCLUDE_COUNT))
 				{
-					tag.setCount(0);
+					tag.setCount(1);
```

Comment:
This looks wrong to me. Either way please add a comment to explain why the count is being set to 1 or to 0.

Reply:
The reason we have changed count from 0 to 1 as we are adding a tag here with noderef.

Reply:
This is wrong - this method creates orphan tags, which are not related with any content. This should be 0.